h/views.py

```diff
@@ -150,15 +157,10 @@ def stream_atom(request):
     if params["limit"] > max_limit:
         params["limit"] = max_limit
 
-    try:
-        annotations = request.api_client.get(
-            "/search", params=params)["rows"]
-    except api_client.ConnectionError as err:
-        raise httpexceptions.HTTPServiceUnavailable(err)
-    except api_client.Timeout as err:
-        raise httpexceptions.HTTPGatewayTimeout(err)
-    except api_client.APIError as err:
-        raise httpexceptions.HTTPBadGateway(err)
+    annotations = api.search_annotations(
```

Comment:
Rather than factoring out things into a logic function maybe just call the view from `h.api.views` directly.